apps/mobile-alphabet/src/app/styles.ts

```diff
@@ -0,0 +1,65 @@
+import { StyleSheet } from 'react-native';
```

Comment:
Interesting how many different approaches to formatting there are between react and react-native.  be nice if they could just pick one a and go with it, lol!